coq-coqoban.opam

```diff
@@ -15,8 +18,7 @@ game."""
 build: [make "-j%{jobs}%" ]
 install: [make "install"]
 depends: [
-  "ocaml" {>= "4.05.0"}
-  "coq" {(>= "8.10") | (= "dev")}
+  "coq" {>= "8.10"}
```

Comment:
Wouldn't it be better to keep the `| (= "dev")` constraint?

Indeed, it seemed to me it was necessary for testing with `coqorg/coq:dev`;

but I'm a bit puzzled to see that the corresponding job was green… do you have an explanation?

Reply:
Since opam is based on `dpkg` style version comparisons, you need an 8.XX upper bound for the `| (= "dev")` part to be meaningful - otherwise it's redundant. When we only have `>= "8.10"`, `coq.dev` is still allowed:
```shell
dpkg --compare-versions 8.10 lt dev && echo true
true
```

Reply:
OK! I think I figured out what happens there:

it appears `"dev" >= "8.10"`, so the or clause can be removed.

But that wouldn't be the case if, e.g., `coq-coqoban` required `{(>= "8.10") & (< "8.14~")}`.

Reply:
@palmskog, sorry for some reason GitHub didn't show me your previous message before I replied myself 😅